### PR TITLE
ci: nx cloud integration

### PR DESCRIPTION
**Pull request by IKatsuba**

### CODE REVIEW

**Review by MarsiBarsi — APPROVED** [review]
# Well done

File: angular.json [hunk 1]
```diff
@@ -111,18 +116,6 @@
             "sourceRoot": "projects/taiga-schematics",
             "projectType": "library",
             "architect": {
-                "build": {
```
Comment (vladimirpotekhin) [inline angular.json:114]: why did you delete this? :thinking:

**Comment by IKatsuba** [comment]
> why did you delete this? :thinking:  

Ouch. Sorry!

File: angular.json (hunk 1 above)
Comment (IKatsuba) [reply angular.json:114]: Ouch. Sorry!

**Review by vladimirpotekhin — APPROVED** [review]
:rocket:
